Makefile

```
@@ -148,12 +148,12 @@ run-local: build
 .PHONY: generate-queries-docs
 generate-queries-docs: ## generate queries catalog md files
 	$(call print-target)
-	@pip3 install -r .github/generators/requirements.txt
-	@python3 -u .github/generators/docs_generator.py \
+	@pip3 install -r .github/scripts/docs-generator/requirements.txt
+	@python3 -u .github/scripts/docs-generator/docs_generator.py \
```

kindly use the following suggestion:

```suggestion
	@python3 -u .github/scripts/docs-generator/docs-generator.py \
```

---

Thank you. Fixed！